### DISCUSSION TITLE
用 ChatGPT 打造虛擬小鎮

### DISCUSSION THREAD

**Discussion by juliaouo** [1101-GS4719-C/Python-Community-GS4719-C#1390] [科技新知]
一群史丹佛大學和Google的研究人員想來實驗看看是否能夠透過AI來模擬人類的互動行為，他們以ChatGPT創造了25個獨特角色，這些角色以生成代理的形式存在，分別擁有不同姓名、職業、身份、人生目標、行為模式等設定，實驗中，25個AI角色被放到一個像是[《模擬市民》](https://www.vivepostwave.com/65/vr-game-space/)般的虛擬小鎮中，研究人員也發現這些AI角色不僅能夠進行日常生活的自然語言對話與互動，還能建立豐富且複雜的社交關係。

這真是一個非常有趣和有意義的研究項目，因為它允許研究人員觀察人工智能如何模擬人類之間的社交互動，並探討可能影響其行為的因素。
在這個虛擬小鎮中，觀察25個AI居民之間的互動，我們可以看到這些AI居民是如何建立關係、交流、合作和分歧的。通過觀察這些居民的互動，我們可以看到他們之間的社交規則和慣例，以及他們如何處理各種情況和問題。

我們可以觀察到AI居民在不同場合的行為和態度，例如文章中的例子：他們會互相討論政治，並且能夠用得知的訊息繼續與下一個AI居民討論，使之成為熱門話題，並且他們對於此事根據身分、職業等的不同具有不同的態度，已經非常像人類社會會真實發生的事件了！

他們除了依據預設指令行動外，還能夠擁有自發行為，我認為這個實驗真的非常的有趣，並且可以運用在遊戲的 NPC 上，之後的遊戲 NPC 並不是只有幾個預設事件，事件的發展有千萬種可能，想想真是感到非常興奮，這樣又何不是另一個真實世界呢？

參考資料：
[研究人員打造AI虛擬小鎮，觀察25個AI居民之間如何生活、聊天以及辦派對](https://dq.yam.com/post/15346)

**Comment by lyien1021** [comment]
這讓我想到之前我有發過的AI偶像，我也很好奇之後會不會有AI偶像的綜藝，蠻想看AI之間會如何互動，感覺會像看動漫一樣嗎？期待以後有綜藝團隊計畫

**Reply by a3240281370** [reply]
許多AI偶像都好逼真，甚至還有經營社群，有時候真的會分辨不太出來

**Comment by a3240281370** [comment]
其實有一點元宇宙的即視感，而現在真的有虛擬世界平台結合元宇宙，但目前看起來應該還是缺少互動的部分，或許未來更多的結合和應用後真的能發展出另一個有趣的世界！
相關文章：https://web3plus.bnext.com.tw/article/528?

**Comment by ThanatosJun** [comment]
說道 AI 的個性化設定，我之前也看到一個很相似的應用，他依照各類人的談話方式，塑造個性 AI ，讓我們可以選擇對話的對象，其中當然還有其中的運用。如果未來每一個人的手機都加載了 AI 胚胎，能隨著我們成長，我想這是一個很 cool 的事。
貼下先前看到的官網網址：[character.ai](https://beta.character.ai/)
